### ISSUE TITLE
"Replace" function of editor skips every other entry

### ISSUE THREAD

**Issue by spyder-issues-migrator** [spyder-ide/spyder#1121]
_From [philipp....@googlemail.com](https://code.google.com/u/106243277547401574018/) on 2012-07-25T02:44:05Z_

Spyder Version:  2.2.0dev
Python Version:  2.7.3
Qt Version:      4.8.1, PyQt4 (API v1) 4.9.1 on Linux

1. Open spyder
2. create empty file
3. write a bunch of 0s in the file
4. hit Ctrl-H (Replace)
5. Enter 0 in the find and 1 in the replace field on the find & replace bar
6. Click on the "Replace/find" or "->" button. Spyder will always skip one 0 and replace the next one
7. Also works with "replace all" option, which leaves a 01010101 pattern
7. the "<-" button works nicely

_Original issue: http://code.google.com/p/spyderlib/issues/detail?id=1121_

**Comment by spyder-issues-migrator** [comment]
_From [pierre.raybaut](https://code.google.com/u/pierre.raybaut/) on 2012-07-28T13:17:41Z_

Can't reproduce this behavior.

Anyone else?

**Comment by spyder-issues-migrator** [comment]
_From [philipp....@googlemail.com](https://code.google.com/u/106243277547401574018/) on 2012-07-30T05:45:11Z_

I just updated the sources and rebuilt spyder, now the bug is gone. 
I rebuilt it before to check if the bug is persistent and it was. However, when I revert to the old sources I no longer have the bug. Maybe some files were corrupted after the pull and led to this strange bug.
Anyways, bug gone, issue can be closed.
